### ISSUE TITLE
arm64 e.g. Pi4 ?

### ISSUE THREAD

**Issue by hugeps** [abalki001/mariadb-operator#12]
Is it build-able on ARM64?  I've tried dumb installing, but operator container fails to start.

**Comment by igoooor** [comment]
yes please, I am facing the same issue on my macbook m1 (arm chip)
